README.md

```diff
@@ -10,127 +10,99 @@
 
-## How to set up
+## Setup
 
-To set up the bot it was made as simple as possible.
+1. **Configure `config.json`**:
+   - Set your bot's prefix and invite link in the `config.json` file.
+   - Here is an explanation of what everything is:
+    
+      
+      | Variable                  | What it is                                     |
+      | ------------------------- | ---------------------------------------------- |
+      | YOUR_BOT_PREFIX_HERE      | The prefix you want to use for normal commands |
+      | YOUR_BOT_INVITE_LINK_HERE | The link to invite the bot                     |
+      
 
-### `config.json` file
+2. **Set the Bot Token**:
+   - Create an environment variable named `TOKEN` or edit the `.env` file with your bot token.
+   - To set up the token you will have to either make use of the [`.env.example`](.env.example) file, either copy or rename it to `.env` and replace `YOUR_BOT_TOKEN_HERE` with your bot's token.
 
-There is [`config.json`](config.json) file where you can put the
-needed things to edit.
+3. **Install Requirements**:
+   - Run `python -m pip install -r requirements.txt` to install the required packages.
 
-Here is an explanation of what everything is:
+4. **Start the Bot**:
+   - Run `python bot.py` to launch your bot.
 
-| Variable                  | What it is                                     |
-| ------------------------- | ---------------------------------------------- |
-| YOUR_BOT_PREFIX_HERE      | The prefix you want to use for normal commands |
-| YOUR_BOT_INVITE_LINK_HERE | The link to invite the bot                     |
+## Support
 
-### `.env` file
+If you need help or have questions, feel free to join our [Discord server](https://discord.com/invite/mTBrXyWxAF) for assistance.
 
-To set up the token you will have to either make use of the [`.env.example`](.env.example) file, either copy or rename it to `.env` and replace `YOUR_BOT_TOKEN_HERE` with your bot's token.
 
-Alternatively you can simply create an environment variable named `TOKEN`.
+## Versioning
 
-## How to start
+We use [SemVer](https://semver.org/) for versioning. Check the repository's tags for available versions.
 
-To start the bot you simply need to launch, either your terminal (Linux, Mac & Windows), or your Command Prompt (
-Windows)
-.
+## Built With
 
-Before running the bot you will need to install all the requirements with this command:
+- Python 3.11.5
 
-```
-python -m pip install -r requirements.txt
-```
+## Disclaimer
 
-After that you can start it with
+Slash commands can take some time to get registered globally, so if you want to test a command you should use
+the `@app_commands.guilds()` decorator so that it gets registered instantly. Example:
 
-```
-python bot.py
+```py
+@commands.hybrid_command(
+  name="command",
+  description="Command description",
+)
+@app_commands.guilds(discord.Object(id=GUILD_ID)) # Place your guild ID here
 ```
```

Comment:
Not sure if we should keep that, slash commands don't really need as long as they used to, to get registered - nowadays it's pretty much instant.